symengine/lib/symengine_wrapper.pyx

```diff
@@ -4560,6 +4560,31 @@ cdef class _Lambdify(object):
             return result
 
 
+cdef double _scipy_callback_lambda_real(int n, double *x, void *user_data) nogil:
+    cdef symengine.LambdaRealDoubleVisitor* lamb = <symengine.LambdaRealDoubleVisitor *>user_data
+    cdef double result
+    deref(lamb).call(&result, x)
+    return result
+
+
+IF HAVE_SYMENGINE_LLVM:
+    cdef double _scipy_callback_llvm_real(int n, double *x, void *user_data) nogil:
+        cdef symengine.LLVMDoubleVisitor* lamb = <symengine.LLVMDoubleVisitor *>user_data
+        cdef double result
+        deref(lamb).call(&result, x)
+        return result
+
+
+def create_low_level_callable(lambdify, *args):
+    from scipy import LowLevelCallable
+    class LambdifyLowLevelCallable(LowLevelCallable):
```

Comment:
Does this class need to be closure? Or could it be defined in a try/except block around `from scipy import LowLevelCallable`?

Reply:
I didn't want to add the initialization time of scipy to symengine. That's why this is done inside a function.